### PR TITLE
CBBD-114: Add AWS API credentials to Jenkins, for ITs that need to work with S3

### PR DESCRIPTION
**Pull request by karlmdavis**
http://issues.hhsdevcloud.us/browse/CBBD-114

This is the build side of this. There's also a build *server* pull request, too.

### CODE REVIEW

**Comment by karlmdavis** [comment]
@sbrockhoff and @dho937 Can you guys please review this (very small) PR?

You can see the test results here: http://builds.hhsdevcloud.us/job/HHSIDEAlab/job/bluebutton-data-pipeline/job/cbbd-114-aws-credentials-build/lastCompletedBuild/testReport/gov.hhs.cms.bluebutton.datapipeline.rif.extract.s3/

File: dev/devenv-readme.md [hunk 1]
```diff
@@ -32,3 +34,9 @@ Unfortunately, this project will sometimes need to depend on an interim/snapshot
     $ mvn clean install -DskipITs=true -DskipTests=true
 
 Once the build is done, the HAPI FHIR artifacts will be placed into your user's local Maven repository (`~/.m2/repository`), available for use by this project or others.
+
+### AWS Credentials
+
+Some of this project's integration tests require AWS credentials. These credentials are used to stand up and tear down AWS resources used in the tests. For example, `DataSetMonitorIT` uses the AWS API to create and manipulate the S3 buckets that it uses during the tests, removing them at the end of each test case. This test code uses the AWS Java API's [DefaultAWSCredentialsProviderChain](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html) class, which will supports multiple different mechanisms for retrieving the credentials to use. Credentials must be supplied via at least one of those mechanisms, or the ITs will fail.
+
+(The project's `Jenkinsfile` uses the environment variable mechanism, which [the project's main Jenkins server](http://builds.hhsdevcloud.us/) has been configured to support.)
```
Comment (sbrockhoff) [inline dev/devenv-readme.md]: Could specify what environment variables are used here to save a click

**Comment by sbrockhoff** [comment]
:+1:

File: dev/devenv-readme.md (hunk 1 above)
Comment (karlmdavis) [inline dev/devenv-readme.md]: I think I'll pass, as folks actually really should crack that open and look at the list of chained providers. Per conversation with Mike this morning, we'll likely end up using env vars during local dev & test, and the instance profile in the HealthAPT dev/test/prod systems.

**Pull request merged**